### ISSUE TITLE
Manual hex color input wrongly changes red and blue

### ISSUE THREAD

**Issue by zorbakus** [Open-Shell/Open-Shell-Menu#82]
For example, if I set taskbar color to FF0000, it becomes blue instead of red. The first two and last two digits are switched in the final output.

**Comment by CTVCAM8** [comment]
Other info of interest

https://ofekshilon.com/2012/02/04/rgb-vs-bgr-in-bitmaps/

https://stackoverflow.com/questions/8773648/explanation-of-an-rgb-to-bgr-c-macro

**Comment by ibuprophen1** [comment]
I believe that the taskbar uses the $SystemAccentDark2 or $SystemAccentDark3 color depending if it is transparent or not. You can try changing the menu to
Main_bitmap=$SystemAccentDark2 

Regarding the the colors for this software, I believe that they're expressed in specific Hex values.

One popular place to look-up the hex values for the color you want is to use "https://www.w3schools.com/colors/colors_picker.asp" and make sure the text color and background color are not the same, otherwise the text will be unreadable.
 
I hope that I had helped to clarify this better via text... :-) 

Good Luck! 


~Ibuprophen

**Comment by CTVCAM8** [comment]
![override color](https://user-images.githubusercontent.com/44127216/46922698-e3114e00-cfc1-11e8-9eb5-0f0c976db9d5.png)
Hello Ibuprophen
As some of us are not programmers, we may not describe requests in the exact fashion required here. :)

zorbakus is describing what happens when 'taskbar color' is changed with a hex value entered into the shell settings.
This overrides the Windows taskbar color.
zorbakus is asking for the hex number to follow the standard format for most programs. 
RRGGBB not BBGGRR.

It's the same issue with menu 'Overide glass color'.
If one wants to copy a hex color from another program, say Adobe, or 3schools link,   
and paste that into taskbar color it turns opposite of what is to be expected.
FF0000 which is red on web page will now became blue in the shell program. 

From what I gather from the links I supplied, looks like Ivo used 
'COLORREF' 0x00bbggrr instead of 'RGBQUAD'
It would be swell if RGBQUAD or BGR to RGB C++ macro be incorporated into this shell program so everthing jives.

**Comment by ibuprophen1** [comment]
I believe that the color should be entered as "#FF0000"... 


~Ibuprophen

**Comment by ibuprophen1** [comment]
I do know that there's a handful of other circumstances that can result in this. I do apologize for providing possible solutions that reflects this one at a time. 

I do know that the transparency feature being used as well can also affect the results of a color. 


~Ibuprophen

**Comment by CTVCAM8** [comment]
Yes, transparency, glass blur, acrylic blur complicates further due to the mixing of desktop colors behind the taskbar/menu. What zorbakus and I are talking about is the basic opaque color. 


I, aka juniper7 had mentioned similar issue about inputting a hex number in The Classic Shell forum way back

http://classicshell.net/forum/viewtopic.php?f=7&t=2666&hilit=colors+in+RRGGBB

It just kind of bugs me that separate color pickers example: Color Cop, NZ software Hex color finder can have it work as photo editors do. I suspect due to the complexity of the shell menu program, changing one thing could mess up a lot of other things and would not be worth it for the few people like zorbakus and me who actually use color override features.
Then again hey, it's not just one developer with Open Shell. Now it's the whole world of developers. :)

**Comment by scw2wi** [comment]
When I'm installing the SW on Windows 11, the default color 141DC2 is shown as red.
I think this is not intended and looks similar like this old reported bug.
And the next thing is, changing the color in the settings does not change the menu color at all.

**Comment by scw2wi** [comment]
Also with Version 4.4.170 I'm still having the same problem under Windows 11,
that the "Menu Look / Override glass color" does not change the Menu glass color.
Whatever I'm selecting, the menu is red in all the cases.
Is this problem or any solution known?
